.rubocop.yml

```diff
@@ -1,3 +1,4 @@
+require: rubocop-rails
 AllCops:
   Include:
     - '**/Rakefile'
```

Comment:
I realized that the original rubocop did not include a lot of the ruby files (because there were a lot of ruby issues in them). Could you add the paths to all the ruby files in app/controllers and app/models?